modules/ROOT/nav.adoc

```diff
@@ -1,9 +1,10 @@
 * xref:index.adoc[Overview]
-* Serverless Eventing
-** Using Eventing with OpenShift Service Mesh
-*** xref:serverless-eventing:service-mesh/eventing-service-mesh-setup.adoc[Setup Eventing with OpenShift Service Mesh]
-*** xref:serverless-eventing:service-mesh/eventing-service-mesh-containersource.adoc[Using ContainerSource with OpenShift Service Mesh]
-*** xref:serverless-eventing:service-mesh/eventing-service-mesh-sinkbinding.adoc[Using SinkBinding with OpenShift Service Mesh]
+* Serverless Common

```

Comment:
Good points. Maybe we should discuss this with docs when they start to integrate that content into the official docs. All of this might probably end up in "Integrations" -> "Service Mesh" in the official docs for now.

Reply:
+1, I'm in favour to keep that "Integration" -> "Service Mesh" path